etc/inc/notices.inc

```diff
@@ -350,6 +350,10 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 	if ($config['notifications']['smtp']['fromaddress']) {
 		$from = $config['notifications']['smtp']['fromaddress'];
+		preg_match('/[\w\s]+/', $from, $matches);
+		$from_display = trim($matches[0]);
+		if(preg_match('/(?:<)(.+)(?:>)$/', $from, $matches))
+			$from = $matches[1];
```

Comment:
Based on this logic, if user fills 'fromaddress' with "renato@pfsense.org" it will end up with:
```
From: renato <renato@pfsense.org>
```
Where is the current code preventing you to fill the fromaddress field with an address in RFC822 format? This change is only forcing and changing user input.